--- wtmpdb-0.74.0/meson.build.orig	2025-04-24 11:54:18.000000000 +0200
+++ wtmpdb-0.74.0/meson.build	2025-08-26 21:42:15.192904385 +0200
@@ -88,7 +88,7 @@ if prefixdir != rootprefixdir and rootpr
         error('Prefix is not below root prefix (now rootprefix=@0@ prefix=@1@)'.format(rootprefixdir, prefixdir))
 endif
 libexecdir = join_paths(prefixdir, get_option('libexecdir'))
-systemunitdir = prefixdir / 'lib/systemd/system'
+systemunitdir = '/lib/systemd/system'
 tmpfilesdir = prefixdir / 'lib/tmpfiles.d'
 
 libpam = cc.find_library('pam')
